tools/update-prod-static

```diff
@@ -85,25 +89,29 @@ run(['./manage.py', 'collectstatic', '--no-default-ignore',
 
 if not settings.PRODUCTION:
     # When building a release tarball, we need to move staticfiles.json
-    run(['mv', 'prod-static/serve/staticfiles.json', 'staticfiles.json'],
-        stdout=fp, stderr=fp)
+    shutil.move('prod-static/serve/staticfiles.json', 'staticfiles.json')
 
 # Compile translation strings to generate `.mo` files.
 run(['./manage.py', 'compilemessages'], stdout=fp, stderr=fp)
 
+# Needed if PRODUCTION
+os.makedirs('prod-static', exist_ok=True)
+
 # Move the source maps out of the serve/ directory and into their
 # proper place.
-run(['rm', '-rf', 'prod-static/source-map'], stdout=fp, stderr=fp)
+if os.path.exists('prod-static/source-map'):
+    shutil.rmtree('prod-static/source-map')
 
-# Needed if PRODUCTION
-run(['mkdir', '-p', 'prod-static'], stdout=fp, stderr=fp)
-
-run(['mv', os.path.join(settings.STATIC_ROOT, 'source-map'), 'prod-static/source-map'],
-    stdout=fp, stderr=fp)
+shutil.move(os.path.join(settings.STATIC_ROOT, 'source-map'), 'prod-static/source-map')
 
 # Move language_options.json to the production release
-run(['cp', '-aT', 'static/locale', os.path.join(settings.STATIC_ROOT, 'locale')],
-    stdout=fp, stderr=fp)
+for dirpath, dirnames, filenames in os.walk('static/locale'):
+    dirpath = os.path.relpath(dirpath, 'static')
+    os.makedirs(os.path.join(settings.STATIC_ROOT, dirpath), exist_ok=True)
+    for filename in filenames:
+        shutil.copy2(
+            os.path.join('static', dirpath, filename),
+            os.path.join(settings.STATIC_ROOT, dirpath, filename))
```

Comment:
Same thought with this; I prefer the `cp` version, because it's more readable, and the perf is not going to be different (`cp` is fast).